SharedPackages/BrowserServicesKit/Sources/Subscription/Flows/Stripe/StripePurchaseFlowV2.swift

```diff
@@ -112,20 +112,30 @@ public final class DefaultStripePurchaseFlowV2: StripePurchaseFlowV2 {
                                               availableEntitlements: features))
     }
 
-    public func subscriptionTierOptions() async -> Result<SubscriptionTierOptions, StripePurchaseFlowError> {
-        Logger.subscriptionStripePurchaseFlow.log("[StripePurchaseFlowV2] Getting subscription tier options for Stripe")
+    public func subscriptionTierOptions(includeProTier: Bool) async -> Result<SubscriptionTierOptions, StripePurchaseFlowError> {
+        Logger.subscriptionStripePurchaseFlow.log("[StripePurchaseFlowV2] Getting subscription tier options for Stripe (includeProTier: \(includeProTier))")
 
-        let regionParameter: String? = isUSRegion() ? "US" : "ROW"
+        let regionParameter: String? = isUSRegion() ? "us" : "row"
```

Comment:
Also as discussed on previous PR, I would be very hesitant in implementing some new custom logic for determining region for Stripe purchases. For now I would stick to hardcoding that Stripe purchase flow uses fixed US region.